opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -352,6 +353,8 @@ namespace Opm
                          SimulationDataContainer& simulatorstate,
                          WellStateFullyImplicitBlackoil& wellstate)
     {
+        // gives a dummy dynamic_list_econ_limited
+        DynamicListEconLimited dummy_list_econ_limited;
```

Comment:
That begs the question: will a list of closed wells/connections due to econ limits be written to the restart file? In any case, I assume that is not supported by out I/O facilities now. @joakim-hove do you know?